src/pages/books.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
+import BookCard from '../components/BookCard';
+import { IBook } from '../interfaces/books';
+
+const Books = () => {
+    const [books, setBooks] = useState<IBook[]>([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    const fetchBookData = async () => {
+        try {
+            const response = await fetch('/mockData/books.json');
+            const json = await response.json();
+            setBooks(json);
+        } catch (error) {
+            console.error('Error fetching data:', error);
```

Comment:
Are we fine with having a page with no content/message in case there is an error?

Reply:
Yes, good for now